.mergify.yml

```diff
@@ -1,21 +1,17 @@
 queue_rules:
   - name: default
-    conditions:
-      - "#approved-reviews-by>1"
-
-pull_request_rules:
-  - name: automerge to main with label automerge and branch protection passing
-    conditions:
+    queue_conditions:
       - "#approved-reviews-by>1"
       - base=main
       - label=A:automerge
-    actions:
-      queue:
-        name: default
-        method: squash
-        commit_message_template: |
-          {{ title }} (#{{ number }})
-          {{ body }}
+    merge_conditions:
+      - "#approved-reviews-by>1"
+    merge_method: squash
+    commit_message_template: |
+      {{ title }} (#{{ number }})
+      {{ body }}
```

Comment:
this looks suspect because this was moved from under the automerge thing to the queue rules.

Since GitHub now has automerge, we don't use automerge from mergify so I'm inclined to delete that setting entirely.